src/services/plan-service.test.js

```diff
@@ -341,7 +341,7 @@ describe('Doppler plan client', () => {
     var plans = await planService.getPlans(currentPlan, 'standard', 'subscribers', planList);
```

Comment:
```suggestion
const plans = await planService.getPlans(currentPlan, 'standard', 'subscribers', planList);
```

Reply:
Ok,  @jcamposmk  good catch! this test file uses `var` in many places, when it declares `var plans = await planService.getPlans` . What do you think about change `var` for `const ` in all that places?

Reply:
Changes applied @jcamposmk  😉